itou/templates/job_seekers_views/nir_modification_request.html

```diff
@@ -0,0 +1,49 @@
+{% extends "layout/base.html" %}
+{% load buttons_form %}
+{% load components %}
+{% load django_bootstrap5 %}
+{% load matomo %}
+
+{% block title %}
+    Demande de régularisation NIR
+    {% if is_request_from_proxy %}de {{ job_seeker.get_full_name }}{% endif %}
+    {{ block.super }}
+{% endblock %}
+
+{% block title_navinfo %}
+    {% include "layout/previous_step.html" with back_url=back_url only %}
+{% endblock %}
+
+{% block title_content %}
+    {% component_title c_title__main=c_title__main role="group" aria-label="Actions sur le candidat" %}
+        {% fragment as c_title__main %}
+            <h1>
+                Demande de régularisation NIR
+                {% if is_request_from_proxy %}de {{ job_seeker.get_full_name }}{% endif %}
+            </h1>
+        {% endfragment %}
+    {% endcomponent_title %}
+{% endblock %}
+
+{% block content %}
+    <section class="s-section">
+        <div class="s-section__container container">
+            <div class="s-section__row row">
+                <div class="s-section__col col-12">
+                    <div class="row">
+                        <div class="col-12">
+                            <div class="c-box mb-3 mb-md-4">
+                                <form method="post" class="js-prevent-multiple-submit">
+                                    {% csrf_token %}
+                                    {% bootstrap_field form.new_nir %}
+                                    {% itou_buttons_form primary_label="Enregistrer et quitter" reset_url=back_url %}
+                                </form>
+                            </div>
+                        </div>
+                    </div>
```

Comment:
```suggestion
        <div class="c-box mb-3 mb-md-4">
            <form method="post" class="js-prevent-multiple-submit">
                {% csrf_token %}
                {% bootstrap_field form.new_nir %}
                {% itou_buttons_form primary_label="Enregistrer et quitter" reset_url=back_url %}
            </form>
        </div>
```

Ce `.row + .col-12` est inutile. Autant que possible, il est préférable d'éviter la "divite" et de garder un DOM sans nœud inutile.